src/helpers/guest.js

```diff
@@ -0,0 +1,19 @@
+import {guest} from "../server/models";
+import asyncHanlder from "express-async-handler";
```

Comment:
why this?

Reply:
asyncHandler handles the asynchronous operations. But I have removed it. Async alone will handle it.